{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/tests.py

```diff
@@ -85,3 +89,53 @@ def test_password_reset(test_user, client):
 @pytest.mark.django_db
 def test_user_token_gets_created_from_signal(test_user):
     assert test_user.auth_token
+
+
+@pytest.mark.django_db
+class TestPreviewTemplateView:
+    url = "/api/template_preview/"
+
+    @override_settings(DEBUG=False)
+    def test_disabled_if_not_debug(self, client):
+        response = client.post(self.url)
+        assert response.status_code == 404
+
+    @override_settings(DEBUG=True)
+    def test_enabled_if_debug(self, client):
```

Comment:
Why do this? Why does it matter if it is in Debug or not?

Reply:
I thought it would be wise to disable this view in production / production-like envs. It is only intended to ease development, so we do not actually need it in other environments

https://github.com/thinknimble/tn-spa-bootstrapper/pull/193/files#diff-8b5450c1f1b12638e18ae360a56f149bce98e774720b01f226cf1b8129a9f893R139-R140